From 5b1259f53fdc561d416ac38faa0e0d65ae69178e Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 9 May 2014 13:49:31 +0800
Subject: [PATCH 317/602] ENGR00304825 Camera apps are not same as in shotcut
 and main menu

Remove the overlay for auto and sd board, since default camera2 used
Update the overlay for sl evk, to add for camera package

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 .../res/xml-sw600dp/default_workspace.xml          |  8 +-
 .../res/xml-sw720dp/default_workspace.xml          |  4 +-
 .../apps/Launcher2/res/xml/default_workspace.xml   |  8 +-
 .../res/xml-sw600dp/default_workspace.xml          | 91 ----------------------
 .../res/xml-sw720dp/default_workspace.xml          | 61 ---------------
 .../apps/Launcher2/res/xml/default_workspace.xml   | 91 ----------------------
 .../res/xml-sw600dp/default_workspace.xml          | 91 ----------------------
 .../res/xml-sw720dp/default_workspace.xml          | 61 ---------------
 .../apps/Launcher2/res/xml/default_workspace.xml   | 91 ----------------------
 .../res/xml-sw600dp/default_workspace.xml          |  8 +-
 .../res/xml-sw720dp/default_workspace.xml          |  4 +-
 .../apps/Launcher2/res/xml/default_workspace.xml   |  8 +-
 12 files changed, 20 insertions(+), 506 deletions(-)
 delete mode 100644 sabreauto_6q/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
 delete mode 100644 sabreauto_6q/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
 delete mode 100644 sabreauto_6q/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
 delete mode 100644 sabresd_6dq/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
 delete mode 100644 sabresd_6dq/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
 delete mode 100644 sabresd_6dq/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml

diff --git a/evk_6sl/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml b/evk_6sl/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
index b5bb7cd..0537169 100755
--- a/evk_6sl/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
+++ b/evk_6sl/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
@@ -37,8 +37,8 @@
         launcher:spanX="2"
         launcher:spanY="2" />
     <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
+        launcher:packageName="com.android.camera"
+        launcher:className="com.android.camera.Camera"
         launcher:screen="2"
         launcher:x="0"
         launcher:y="3" />
@@ -61,8 +61,8 @@
 
     <!-- Hotseat (We use the screen as the position of the item in the hotseat) -->
     <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.DialtactsActivity"
+        launcher:packageName="com.android.dialer"
+        launcher:className="com.android.dialer.DialtactsActivity"
         launcher:container="-101"
         launcher:screen="1"
         launcher:x="1"
diff --git a/evk_6sl/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml b/evk_6sl/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
index 1defd4c..6302d7e 100755
--- a/evk_6sl/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
+++ b/evk_6sl/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
@@ -37,8 +37,8 @@
         launcher:spanX="2"
         launcher:spanY="2" />
     <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
+        launcher:packageName="com.android.camera"
+        launcher:className="com.android.camera.Camera"
         launcher:screen="2"
         launcher:x="0"
         launcher:y="3" />
diff --git a/evk_6sl/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml b/evk_6sl/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
index 86a2a69..93338d3 100755
--- a/evk_6sl/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
+++ b/evk_6sl/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
@@ -37,8 +37,8 @@
         launcher:spanX="2"
         launcher:spanY="2" />
     <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
+        launcher:packageName="com.android.camera"
+        launcher:className="com.android.camera.Camera"
         launcher:screen="2"
         launcher:x="0"
         launcher:y="3" />
@@ -61,8 +61,8 @@
 
     <!-- Hotseat (We use the screen as the position of the item in the hotseat) -->
     <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.DialtactsActivity"
+        launcher:packageName="com.android.dialer"
+        launcher:className="com.android.dialer.DialtactsActivity"
         launcher:container="-101"
         launcher:screen="0"
         launcher:x="0"
diff --git a/sabreauto_6q/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml b/sabreauto_6q/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
deleted file mode 100644
index b5bb7cd..0000000
--- a/sabreauto_6q/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
+++ /dev/null
@@ -1,91 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2012 The Android Open Source Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<favorites xmlns:launcher="http://schemas.android.com/apk/res/com.android.launcher">
-    <!-- Far-left screen [0] -->
-
-    <!-- Left screen [1] -->
-    <appwidget
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.widget.SettingsAppWidgetProvider"
-        launcher:screen="1"
-        launcher:x="0"
-        launcher:y="3"
-        launcher:spanX="4"
-        launcher:spanY="1" />
-
-    <!-- Middle screen [2] -->
-    <appwidget
-        launcher:packageName="com.android.deskclock"
-        launcher:className="com.android.alarmclock.AnalogAppWidgetProvider"
-        launcher:screen="2"
-        launcher:x="1"
-        launcher:y="0"
-        launcher:spanX="2"
-        launcher:spanY="2" />
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
-        launcher:screen="2"
-        launcher:x="0"
-        launcher:y="3" />
-
-    <!-- Right screen [3] -->
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.gallery3d.app.Gallery"
-        launcher:screen="3"
-        launcher:x="1"
-        launcher:y="3" />
-    <favorite
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.Settings"
-        launcher:screen="3"
-        launcher:x="2"
-        launcher:y="3" />
-
-    <!-- Far-right screen [4] -->
-
-    <!-- Hotseat (We use the screen as the position of the item in the hotseat) -->
-    <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.DialtactsActivity"
-        launcher:container="-101"
-        launcher:screen="1"
-        launcher:x="1"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.PeopleActivity"
-        launcher:container="-101"
-        launcher:screen="2"
-        launcher:x="2"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.mms"
-        launcher:className="com.android.mms.ui.ConversationList"
-        launcher:container="-101"
-        launcher:screen="4"
-        launcher:x="4"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.browser"
-        launcher:className="com.android.browser.BrowserActivity"
-        launcher:container="-101"
-        launcher:screen="5"
-        launcher:x="5"
-        launcher:y="0" />
-</favorites>
diff --git a/sabreauto_6q/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml b/sabreauto_6q/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
deleted file mode 100644
index 1defd4c..0000000
--- a/sabreauto_6q/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
+++ /dev/null
@@ -1,61 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2009 The Android Open Source Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<favorites xmlns:launcher="http://schemas.android.com/apk/res/com.android.launcher">
-    <!-- Far-left screen [0] -->
-
-    <!-- Left screen [1] -->
-    <appwidget
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.widget.SettingsAppWidgetProvider"
-        launcher:screen="1"
-        launcher:x="0"
-        launcher:y="3"
-        launcher:spanX="4"
-        launcher:spanY="1" />
-
-    <!-- Middle screen [2] -->
-    <appwidget
-        launcher:packageName="com.android.deskclock"
-        launcher:className="com.android.alarmclock.AnalogAppWidgetProvider"
-        launcher:screen="2"
-        launcher:x="1"
-        launcher:y="0"
-        launcher:spanX="2"
-        launcher:spanY="2" />
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
-        launcher:screen="2"
-        launcher:x="0"
-        launcher:y="3" />
-
-    <!-- Right screen [3] -->
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.gallery3d.app.Gallery"
-        launcher:screen="3"
-        launcher:x="1"
-        launcher:y="3" />
-    <favorite
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.Settings"
-        launcher:screen="3"
-        launcher:x="2"
-        launcher:y="3" />
-
-    <!-- Far-right screen [4] -->
-</favorites>
diff --git a/sabreauto_6q/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml b/sabreauto_6q/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
deleted file mode 100644
index 86a2a69..0000000
--- a/sabreauto_6q/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
+++ /dev/null
@@ -1,91 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2009 The Android Open Source Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<favorites xmlns:launcher="http://schemas.android.com/apk/res/com.android.launcher">
-    <!-- Far-left screen [0] -->
-
-    <!-- Left screen [1] -->
-    <appwidget
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.widget.SettingsAppWidgetProvider"
-        launcher:screen="1"
-        launcher:x="0"
-        launcher:y="3"
-        launcher:spanX="4"
-        launcher:spanY="1" />
-
-    <!-- Middle screen [2] -->
-    <appwidget
-        launcher:packageName="com.android.deskclock"
-        launcher:className="com.android.alarmclock.AnalogAppWidgetProvider"
-        launcher:screen="2"
-        launcher:x="1"
-        launcher:y="0"
-        launcher:spanX="2"
-        launcher:spanY="2" />
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
-        launcher:screen="2"
-        launcher:x="0"
-        launcher:y="3" />
-
-    <!-- Right screen [3] -->
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.gallery3d.app.Gallery"
-        launcher:screen="3"
-        launcher:x="1"
-        launcher:y="3" />
-    <favorite
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.Settings"
-        launcher:screen="3"
-        launcher:x="2"
-        launcher:y="3" />
-
-    <!-- Far-right screen [4] -->
-
-    <!-- Hotseat (We use the screen as the position of the item in the hotseat) -->
-    <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.DialtactsActivity"
-        launcher:container="-101"
-        launcher:screen="0"
-        launcher:x="0"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.PeopleActivity"
-        launcher:container="-101"
-        launcher:screen="1"
-        launcher:x="1"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.mms"
-        launcher:className="com.android.mms.ui.ConversationList"
-        launcher:container="-101"
-        launcher:screen="3"
-        launcher:x="3"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.browser"
-        launcher:className="com.android.browser.BrowserActivity"
-        launcher:container="-101"
-        launcher:screen="4"
-        launcher:x="4"
-        launcher:y="0" />
-</favorites>
diff --git a/sabresd_6dq/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml b/sabresd_6dq/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
deleted file mode 100644
index b5bb7cd..0000000
--- a/sabresd_6dq/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
+++ /dev/null
@@ -1,91 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2012 The Android Open Source Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<favorites xmlns:launcher="http://schemas.android.com/apk/res/com.android.launcher">
-    <!-- Far-left screen [0] -->
-
-    <!-- Left screen [1] -->
-    <appwidget
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.widget.SettingsAppWidgetProvider"
-        launcher:screen="1"
-        launcher:x="0"
-        launcher:y="3"
-        launcher:spanX="4"
-        launcher:spanY="1" />
-
-    <!-- Middle screen [2] -->
-    <appwidget
-        launcher:packageName="com.android.deskclock"
-        launcher:className="com.android.alarmclock.AnalogAppWidgetProvider"
-        launcher:screen="2"
-        launcher:x="1"
-        launcher:y="0"
-        launcher:spanX="2"
-        launcher:spanY="2" />
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
-        launcher:screen="2"
-        launcher:x="0"
-        launcher:y="3" />
-
-    <!-- Right screen [3] -->
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.gallery3d.app.Gallery"
-        launcher:screen="3"
-        launcher:x="1"
-        launcher:y="3" />
-    <favorite
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.Settings"
-        launcher:screen="3"
-        launcher:x="2"
-        launcher:y="3" />
-
-    <!-- Far-right screen [4] -->
-
-    <!-- Hotseat (We use the screen as the position of the item in the hotseat) -->
-    <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.DialtactsActivity"
-        launcher:container="-101"
-        launcher:screen="1"
-        launcher:x="1"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.PeopleActivity"
-        launcher:container="-101"
-        launcher:screen="2"
-        launcher:x="2"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.mms"
-        launcher:className="com.android.mms.ui.ConversationList"
-        launcher:container="-101"
-        launcher:screen="4"
-        launcher:x="4"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.browser"
-        launcher:className="com.android.browser.BrowserActivity"
-        launcher:container="-101"
-        launcher:screen="5"
-        launcher:x="5"
-        launcher:y="0" />
-</favorites>
diff --git a/sabresd_6dq/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml b/sabresd_6dq/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
deleted file mode 100644
index 1defd4c..0000000
--- a/sabresd_6dq/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
+++ /dev/null
@@ -1,61 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2009 The Android Open Source Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<favorites xmlns:launcher="http://schemas.android.com/apk/res/com.android.launcher">
-    <!-- Far-left screen [0] -->
-
-    <!-- Left screen [1] -->
-    <appwidget
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.widget.SettingsAppWidgetProvider"
-        launcher:screen="1"
-        launcher:x="0"
-        launcher:y="3"
-        launcher:spanX="4"
-        launcher:spanY="1" />
-
-    <!-- Middle screen [2] -->
-    <appwidget
-        launcher:packageName="com.android.deskclock"
-        launcher:className="com.android.alarmclock.AnalogAppWidgetProvider"
-        launcher:screen="2"
-        launcher:x="1"
-        launcher:y="0"
-        launcher:spanX="2"
-        launcher:spanY="2" />
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
-        launcher:screen="2"
-        launcher:x="0"
-        launcher:y="3" />
-
-    <!-- Right screen [3] -->
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.gallery3d.app.Gallery"
-        launcher:screen="3"
-        launcher:x="1"
-        launcher:y="3" />
-    <favorite
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.Settings"
-        launcher:screen="3"
-        launcher:x="2"
-        launcher:y="3" />
-
-    <!-- Far-right screen [4] -->
-</favorites>
diff --git a/sabresd_6dq/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml b/sabresd_6dq/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
deleted file mode 100644
index 86a2a69..0000000
--- a/sabresd_6dq/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
+++ /dev/null
@@ -1,91 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2009 The Android Open Source Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<favorites xmlns:launcher="http://schemas.android.com/apk/res/com.android.launcher">
-    <!-- Far-left screen [0] -->
-
-    <!-- Left screen [1] -->
-    <appwidget
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.widget.SettingsAppWidgetProvider"
-        launcher:screen="1"
-        launcher:x="0"
-        launcher:y="3"
-        launcher:spanX="4"
-        launcher:spanY="1" />
-
-    <!-- Middle screen [2] -->
-    <appwidget
-        launcher:packageName="com.android.deskclock"
-        launcher:className="com.android.alarmclock.AnalogAppWidgetProvider"
-        launcher:screen="2"
-        launcher:x="1"
-        launcher:y="0"
-        launcher:spanX="2"
-        launcher:spanY="2" />
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
-        launcher:screen="2"
-        launcher:x="0"
-        launcher:y="3" />
-
-    <!-- Right screen [3] -->
-    <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.gallery3d.app.Gallery"
-        launcher:screen="3"
-        launcher:x="1"
-        launcher:y="3" />
-    <favorite
-        launcher:packageName="com.android.settings"
-        launcher:className="com.android.settings.Settings"
-        launcher:screen="3"
-        launcher:x="2"
-        launcher:y="3" />
-
-    <!-- Far-right screen [4] -->
-
-    <!-- Hotseat (We use the screen as the position of the item in the hotseat) -->
-    <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.DialtactsActivity"
-        launcher:container="-101"
-        launcher:screen="0"
-        launcher:x="0"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.PeopleActivity"
-        launcher:container="-101"
-        launcher:screen="1"
-        launcher:x="1"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.mms"
-        launcher:className="com.android.mms.ui.ConversationList"
-        launcher:container="-101"
-        launcher:screen="3"
-        launcher:x="3"
-        launcher:y="0" />
-    <favorite
-        launcher:packageName="com.android.browser"
-        launcher:className="com.android.browser.BrowserActivity"
-        launcher:container="-101"
-        launcher:screen="4"
-        launcher:x="4"
-        launcher:y="0" />
-</favorites>
diff --git a/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml b/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
index b5bb7cd..0537169 100644
--- a/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
+++ b/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml-sw600dp/default_workspace.xml
@@ -37,8 +37,8 @@
         launcher:spanX="2"
         launcher:spanY="2" />
     <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
+        launcher:packageName="com.android.camera"
+        launcher:className="com.android.camera.Camera"
         launcher:screen="2"
         launcher:x="0"
         launcher:y="3" />
@@ -61,8 +61,8 @@
 
     <!-- Hotseat (We use the screen as the position of the item in the hotseat) -->
     <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.DialtactsActivity"
+        launcher:packageName="com.android.dialer"
+        launcher:className="com.android.dialer.DialtactsActivity"
         launcher:container="-101"
         launcher:screen="1"
         launcher:x="1"
diff --git a/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml b/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
index 1defd4c..6302d7e 100644
--- a/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
+++ b/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml-sw720dp/default_workspace.xml
@@ -37,8 +37,8 @@
         launcher:spanX="2"
         launcher:spanY="2" />
     <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
+        launcher:packageName="com.android.camera"
+        launcher:className="com.android.camera.Camera"
         launcher:screen="2"
         launcher:x="0"
         launcher:y="3" />
diff --git a/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml b/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
index 86a2a69..93338d3 100644
--- a/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
+++ b/sabresd_6sx/overlay/packages/apps/Launcher2/res/xml/default_workspace.xml
@@ -37,8 +37,8 @@
         launcher:spanX="2"
         launcher:spanY="2" />
     <favorite
-        launcher:packageName="com.android.gallery3d"
-        launcher:className="com.android.camera.CameraLauncher"
+        launcher:packageName="com.android.camera"
+        launcher:className="com.android.camera.Camera"
         launcher:screen="2"
         launcher:x="0"
         launcher:y="3" />
@@ -61,8 +61,8 @@
 
     <!-- Hotseat (We use the screen as the position of the item in the hotseat) -->
     <favorite
-        launcher:packageName="com.android.contacts"
-        launcher:className="com.android.contacts.activities.DialtactsActivity"
+        launcher:packageName="com.android.dialer"
+        launcher:className="com.android.dialer.DialtactsActivity"
         launcher:container="-101"
         launcher:screen="0"
         launcher:x="0"
-- 
1.8.0

